src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -147,8 +146,6 @@ class ClusterMonitorActor(val cluster: Cluster,
       _ <- dbRef.inTransaction { dataAccess =>
         dataAccess.clusterQuery.setToRunning(cluster.id, publicIp)
       }
-      // Ensure the cluster is ready for proxying by updating the IP -> DNS cache
-      _ <- ensureClusterReadyForProxying(publicIp, clusterStatus)
```

Comment:
Note to self to double-check that removing ^ doesn't introduce a regression.

Reply:
We're no longer trying to cache IPs as soon as hosts become available upon cluster creation so `ensureClusterReadyForProxying` (whose name is a bit misleading) is no longer necessary.